### PR TITLE
Change the image registry

### PR DESCRIPTION
**Pull request by PetarTodorovv**
**Description**
Changing the kyma image registry from `eu.gcr.io` to `europe-docker.pkg.dev`. For the time being the directory of the images will be `dev/incubator/` but when we clarify the process it most probably will be changed to `prod/incubator`

Changes proposed in this pull request:
- ...

**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [x] [N/A] Unit tests
- [x] [N/A] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] [N/A] Mocks are regenerated, using the automated script

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -124,49 +124,49 @@ global:
...
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-schema-migrator
     system_broker:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-system-broker
     certs_setup_job:
```
Comment (dzahariev) [inline chart/compass/values.yaml:172]: This is a corner case. 
The image is missing on the new repo. We need to clarify what will happen with old repo. If it is decommissioned, then we may need to transport this image to the new one.

**Review by dzahariev — COMMENTED** [review]
What about buildpack-golang, it is available at https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/test-infra%2Fbuildpack-golang
Probably we should switch it in Makefile-s.

The following charts also need to be adapted:
- chart/compass/charts/testing/values.yaml
- chart/compass/templates/agent-configuration-job.yaml
- chart/compass/templates/registration-job.yaml
- docs/compass/assets/order-service.yaml (cannot find the same image in prod repo)
- installation/scripts/prom-mtls-patch.sh

**Comment by PetarTodorovv** [comment]
> What about buildpack-golang, it is available at https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/test-infra%2Fbuildpack-golang Probably we should switch it in Makefile-s.
> 
> The following charts also need to be adapted:
> 
> * chart/compass/charts/testing/values.yaml
> * chart/compass/templates/agent-configuration-job.yaml
> * chart/compass/templates/registration-job.yaml
> * docs/compass/assets/order-service.yaml (cannot find the same image in prod repo)
> * installation/scripts/prom-mtls-patch.sh

The buildpack images, cert-setup-job image, and the others will be available in the "old" registry. Only the "PR images" will be removed. About the first ones maybe in the near future, we should consider migrating them.

File: chart/compass/values.yaml (hunk 1 above)
Comment (PetarTodorovv) [reply chart/compass/values.yaml:172]: Check the other comment as well but it will be available in the old registry

**Review by dzahariev — DISMISSED** [review]

**Review by dzahariev — APPROVED** [review]

**Pull request merged**